shared/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@finos/calm-shared",
+    "version": "0.2.1",
```

Comment:
Do we now have multiple versions to keep track of? Or is the whole project on one version?

Reply:
This is something I hadn't really considered, currently `shared` is entirely internal.  Potentially should consider marking the `package.json` private somehow.

Reply:
Do we even _need_ a version beyond the top level one that the entire project uses?

Reply:
What about the following;

- Increment minor version within shared - so that CLI and Shared can have different versions.
- Once this PR is merged,  I'll be updating the CLI `package.json` to refer to an explicit version of `shared` module instead of a file reference ( so that version upgrades are managed and CLI version increments can occur when needed ) 
- Mark `shared` `package.json` to have `"private": true`